flow/scripts/open.tcl

```diff
@@ -60,12 +60,74 @@ proc read_timing {input_file} {
 if {[env_var_equals GUI_TIMING 1]} {
   puts "GUI_TIMING=1 reading timing, takes a little while for large designs..."
   read_timing $input_file
+
+  gui::save_display_controls
+
+  set height [[[ord::get_db_block] getBBox] getDY]
+  set height [ord::dbu_to_microns $height]
+  set resolution [expr $height / 1000]
+
+  # FIXME reenable when there is a way to disable the rendered clock tree
+  #
+  # foreach clock [get_clocks *] {
+  #   if { [llength [get_property $clock sources]] > 0 } {
+  #     set clock_name [get_name $clock]
+  #     save_clocktree_image -clock $clock_name \
+  #         -width 100 -height 100 \
+  #         $::env(OBJECTS_DIR)/dummy.png
+  #     break
+  #   }
+  # }
```

Comment:
Agreed, but how else to precalculate this?

Reply:
Does it actually save time?  Most of the effort is to get sta warmed up.  Drawing the tree should be cheap.

Reply:
Yes, last I checked.

However, I can delete the commented code and if this PR is otherwise good to go, we can square away what we already have a solution for and I can then create a new PR with a narrower concern and some numbers.

A way to precalculate this that does not modify GUI state and also does not write files to disk is needed, which isnt available yet, right?

Reply:
Yes, if there really is a need.  I would be curious to see the time saved.